backend/alexa_tutorial.md

```diff
@@ -0,0 +1,71 @@
+##Steps to building an alexa skill:
+
+###Prerequisites:
+In order to develop an Alexa skill you'll need access to 2 accounts.
+
+1. **aws.amazon.com**: An IAM user with access to the webservices@intrepid.io account
+2. **developer.amazon.com**:  webservices@intrepid.io for amazon developers, this login info is in LastPass.
```

Comment:
@Benuuu I think we're already doing lots of jenkins specific guides, which are just for our org